python/lsst/validate/drp/astromerrmodel.py

```diff
@@ -67,6 +67,8 @@ def astromErrModel(snr, theta=1000, sigmaSys=10, C=1, **kwargs):
 def fitAstromErrModel(snr, dist):
     """Fit model of astrometric error from LSST Overview paper
 
```

Comment:
Maybe put in a colon, or make "LSST overview paper" a sphinx reference? (similarly for photerrmodel below)